.github/workflows/gradle.yml

```
@@ -10,31 +10,31 @@ jobs:
   ci:
     runs-on: ubuntu-lastest
```

```suggestion
      runs-on: ubuntu-latest
```
아마도 이거때문에 job이 제대로 안돌아가는 것 같은데 확인 한 번 부탁드려요!

---

오타가 있었네요!! 감사합니다.